ArduCopter/AP_Arming.cpp

```diff
@@ -596,6 +596,21 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
         return false;
     }
 
+    // Do not arm if EKF lane 1 is not used as primary during arming, when using EK2.
+    const NavEKF2 &ekf2 = AP::ahrs_navekf().get_NavEKF2_const();
+    if (ekf2.currentPrimary() != 0) {
+        check_failed(true, "EKF lane 1 not in use as primary");
```

Comment:
Suggest saying EKF2 for additional info and same below for EKF3

Reply:
Done